server/express/Dockerfile

```diff
@@ -15,26 +11,17 @@ ENV NODE_ENV production
 RUN apk --no-cache -U upgrade && apk add python make g++
 
 # Prepare destination directory and ensure user node owns it
-RUN mkdir -p /home/node/app/dist
-RUN mkdir -p /home/node/app/examples/server/express
+RUN mkdir -p /home/node/app/server/express
 RUN chown -R node:node /home/node/app
 
-# Copy the passport strategy code
-WORKDIR /home/node/app
-COPY --chown=node:node /dist ./dist
-COPY --chown=node:node package*.json ./
-
-# Install libraries as user node
-RUN npm i --only=production
-
 # Set CWD
-WORKDIR /home/node/app/examples/server/express
+WORKDIR /home/node/app/server/express
 
 # Install PM2
 RUN npm i -g pm2
 
 # Copy package.json, package-lock.json and process.yml
-COPY --chown=node:node examples/server/express/. ./
+COPY --chown=node:node . ./
 
 # Switch to user node
 USER node
```

Comment:
I would make it a really simple one like this, cause this example not have any dev dependencies: 
```
ARG PORT=3333

FROM node:14-alpine

# Set node environment to production
ENV NODE_ENV production

WORKDIR /app

# Update the system
RUN apk --no-cache -U upgrade && apk add python make g++

COPY . .

# install only prod dependencies
RUN npm i --only=production

# Install PM2
RUN npm i -g pm2

# Switch to user node
USER node

# Open desired port
EXPOSE ${PORT}

# Use PM2 to run the application as stated in config file
ENTRYPOINT ["pm2-runtime", "./process.yml"] 
```